GettingStarted.md

```diff
@@ -0,0 +1,162 @@
+# Introduction
+Welcome to impress.js! This presentation framework allows you to create stunning presentations with the power of CSS3 transformations. It is inspired by the idea behind [prezi.com](https://prezi.com). 
+**NOTE:** This Guide is not made for you, if you have never written HTML and/or CSS before. Knowing your way around in JavaScript certainly helps, but is not a necessity. You may still continue this tutorial and try to understand what we do as you go. 
+
+Also, if you are an experienced (or sorta experienced user, anyway), you might prefer the [DOCUMENTATION](DOCUMENTATION.md).
+
+# Getting started with impress.js
+## Installation / acquiring the framework
+You first need to decide if you want to use the bleeding edge (most up to date, generally very reliable as well, but may contain more problems) or the stable release. You can use one of the methods mentioned below or the direct download link to only the javascript file. The benefit is, you don't have to find the file in the repository and you can copy the link and directly load the file from the cdn. 
+
+### **Direct links to only impress.js file**
+- V2.0.0: https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js
+- V1.1.0: https://cdn.jsdelivr.net/gh/impress/impress.js@1.1.0/js/impress.js
+- Source: https://cdn.jsdelivr.net/gh/impress/impress.js/js/impress.js
+
+For older versions, please just replace the version number behind the @!
+
+**Loading the script directly from cdn**
+Loading the script from the cdn is quite straight forward. You need to include a script tag in the head of the HTML document or where ever you want to include it.:
+
+```
+<script src="https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js"></script>
+```
+
+Note: in order for this to work, you need to be sure that you have a WiFi connection when giving your presentation! If you are sure, then you may use this variant. But be sure to remove 
+
+```
+<script type="text/javascript" src='./impress.js></script>
+```
+from the below example.
+
+### **Stable release:**
+Head to the releases tab and download the source code as zip or as a tarball. Go ahead and unzip / untar it. You may find the framework under ```/js/impress.js```. Do also include the ```/css/impress-common.css```, if you want to make your life a bit easier.
+
+### **Bleeding Edge:**
+In the github repository, click the "Code"-Button, then click "Download ZIP". Go ahead and unzip / untar it. You may find the framework under ```/js/impress.js```. Do also include the ```/css/impress-common.css```, if you want to make your life a bit easier.
+
+## Setting up the project
+Open up your favorite text-editor / IDE, for example Visual Studio Code, Atom, Notepad ++, ...
+Now, copy the *impress.js* file into the folder you are working in and create a new file called *index.html* and create the basic HTML layout:
```

Comment:
While everything you say is correct technically, it's important to be consistent. In all demo presentations and other guidance, even within this file just a few lines above, the `impress.js` file is in a directory called `js/`. So you should keep that through all examples:

    Copy the folder `js/` into the folder you are working in. It contains the `impress.js` file.